test/functional/controller/controller_test.go

```diff
@@ -21,6 +21,7 @@ const (
 	opAdd operation = iota
 	opUpdate
 	opDelete
+	IMPORTER_DEFAULT_IMAGE = "kubevirt/cdi-importer:latest"
```

Comment:
ditto re common.go

Reply:
same as above, only relevent in this test